db/knex_migrations/2024-04-26-0000-snmp-monitor.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (knex) {
+    return knex.schema
+        .alterTable("monitor", function (table) {
+            table.string("snmp_community_string", 255).defaultTo("public"); // Add snmp_community_string column
+            table.string("snmp_oid").defaultTo(null); // Add oid column
+            table.enum("snmp_version", [ "0", "1", "3" ]).defaultTo("1"); // Add snmp_version column with enum values (0: SNMPv1, 1: SNMPv2c, 3: SNMPv3)
```

Comment:
Why are you storing the version like this?
Seems pretty counterintuitive and like a footgun waiting to happen.

Please revert it to

```suggestion
            table.enum("snmp_version", [ "1", "2c", "3" ]).defaultTo("2c");
```